expan/core/statistics.py

```diff
@@ -89,8 +91,10 @@ def delta(x, y, assume_normal=True, percentiles=[2.5, 97.5],
         mu = _delta_mean(_x, _y)
         # Computing the confidence intervals
         if assume_normal:
-            c_i = normal_sample_difference(x=_x, y=_y, percentiles=percentiles,
-                                           relative=relative)
+            if num_tests:
+                percentiles = [x/num_tests if x < 50.0
+                               else 100-(100-x)/num_tests if x > 50.0 else x for x in percentiles]
```

Comment:
in Python 2, dividing an `int` by an `int` always produces an `int`, please make sure that no integer division happens here

Reply:
yes, makes sense, thank you!

Reply:
as a general point we could discuss dropping Python 2 support all together